### PR TITLE
Add an initial suite of tests

### PR DESCRIPTION
**Pull request by larsks**
This PR adds a basic suite of tests to the project. Coverage of the
API modules is okay, but there's a lot to be done to properly exercise
the CLI.

### CODE REVIEW

File: moc_sprint_tools/cardwrapper.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ def __getattr__(self, k):
 
     def __repr__(self):
         title = self.title.splitlines()[0][:40].strip()
-        return f'<title="{title!r}" priority={self.priority} id={self.id}>'
+        return f'<title="{title}" priority={self.priority} id={self.id}>'
```
Comment (knikolla) [inline moc_sprint_tools/cardwrapper.py:17]: We're not using priority in our workflow yet, right?

File: tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    yield mock_api_object
+
+
+@pytest.fixture
+def runner():
+    return click.testing.CliRunner()
+
+
+def test_shell_no_command(api, runner):
```
Comment (knikolla) [inline tests/test_utils.py:24]: This is also present in test_cli

File: tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+
+def test_shell_command(api, runner):
+    result = runner.invoke(moc_sprint_tools.cli.main, 'shell',
+                           input='print("hello world")\n')
+    assert result.exit_code == 0
+    assert 'hello world' in result.stdout
+
+
+def test_boards(api, runner, projects):
```
Comment (knikolla) [inline tests/test_utils.py:36]: this is also duplicated in test_cli

**Review by knikolla — COMMENTED** [review]
This looks good, minor comments. You're also including #12, which perhaps you don't want.

File: moc_sprint_tools/cardwrapper.py (hunk 1 above)
Comment (larsks) [reply moc_sprint_tools/cardwrapper.py:17]: The `sort-cards-by-priority` task uses it, which right now just ensures that some notes consistently float to the top of the notes column.

File: tests/test_utils.py (hunk 2 above)
Comment (larsks) [reply tests/test_utils.py:24]: It doesn't seem to be. Are you looking at a stale checkout? Or am I just missing something obvious? I am looking at https://github.com/CCI-MOC/sprint-tools/blob/b5177152b1a94c4708bc97546a7eb27ba5796abc/tests/test_cli.py.

**Comment by larsks** [comment]
I was intentionally building this on top of #12. I think that PR can merge as soon as we fix the workflow.

File: tests/test_utils.py (hunk 2 above)
Comment (knikolla) [reply tests/test_utils.py:24]: The name is different but the following two lines are the same https://github.com/CCI-MOC/sprint-tools/blob/b5177152b1a94c4708bc97546a7eb27ba5796abc/tests/test_cli.py#L25

**Comment by larsks** [comment]
I've removed the duplicate tests that @knikolla spotted.

**Pull request merged**